jira.js

```diff
@@ -118,6 +118,12 @@ router.post('/webhook', async (req, res) => {
 						fetchedUser.send('New assignment', { embed });
 					}).catch(console.error);
 			}
+		} else if (req.body.transition && req.body.transition.transitionName === 'Finish') {
+			msg.delete();
+			link.finished = true;
+			link.save();
+		} else if (req.body.transition && req.body.transition.transitionName === 'Send to Ikari') {
+			clipRequest(['youtube', req.body.issue.fields.customfield_10200, req.issue.fields.customfield_10201, req.issue.fields.summary, 'mkv']);
```

Comment:
`clipRequest` has a first parameter(message) that needs to be removed, see deepscan.